sormas-api/src/main/resources/captions.properties

```diff
@@ -281,6 +281,7 @@ CampaignFormData.edit=Edit
 caseCasesList=Cases list
 caseInfrastructureDataChanged=Infrastructure data has changed
 caseCloneCaseWithNewDisease=Generate new case for 
+caseUpdateCaseWithNewDiseaseVariant=Update case disease variant 
```

Comment:
Move this from captions to strings and use `headingUpdateCaseWithNewDiseaseVariant` as property